metadata-ingestion/src/datahub/cli/delete_cli.py

```diff
@@ -320,6 +369,9 @@ def _delete_one_urn(
     soft: bool = False,
     dry_run: bool = False,
     entity_type: str = "dataset",
+    aspect_name: Optional[str] = None,
+    startTimeMillis: Optional[datetime] = None,
```

Comment:
param name `startTimeMillis` is misleading as it is just a datetime object

Reply:
Fixed.